.pre-commit-config.yaml

```
@@ -5,6 +5,7 @@ repos:
   - hooks:
       - id: black
         language_version: python3
+        additional_dependencies: ['click==8.0.4']
```

Had to change this bc/`black` was crashing for me locally. Maybe it's an issue for other people? I'm on a totally new Mac env.

Solution from https://github.com/psf/black/issues/2964#issuecomment-1080974737

---

This is an issue with black that has been resolved in the current release (`22.3.0`), this fix belongs in a different PR to update the black pre-commit rev

---

So no need for this right?
```suggestion
  - hooks:
      - id: black
        language_version: python3
```

---

I can confirm that black 22.3 no longer has this issue. Reverting...

See https://github.com/iterative/dvc/pull/7526 though